Presentation.Web/app/models/Odata/it-system-usage/SystemUsageSensitiveDataLevelDTO.ts

```diff
@@ -0,0 +1,26 @@
+﻿module Kitos.Models.Odata.ItSystemUsage {
+
+    export interface IItSystemUsageSensitiveDataLevelDTO {
+        Id: number;
+
+        ItSystemUsage: Models.ItSystemUsage.IItSystemUsage;
+
+        //Odata maps enums to string https://github.com/OData/WebApi/issues/364
+        SensitivityDataLevel: string;
+    }
+
+    export class SensitiveDataLevelMapper {
+        static map(input: string) {
+            switch (input) {
+                case "NONE":
+                    return "Ingen persondata";
+                case "PERSONALDATA":
```

Comment:
Se kommentaren til gdpr view for konsolidering af sensistivity level modellen.. her vil man så have et lokalt hashmap som metoden her slår op i (enumTextvalue til text map)